### PR TITLE
Update app name, payment status desc in UG

### PR DESCRIPTION
**Pull request by tanwartushar**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -152,7 +153,7 @@ Examples:
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 
-1. The deadline field will show an `[OVERDUE]` tag if the deadline has passed and the client status is still `active`, so if the client has paid, and you're business with them is finished, remember to set their client status to 'old' so the deadline field doesn't show `[OVERDUE]`.
+1. The deadline field will show an `OVERDUE` label if the deadline has passed and the client status is still `active`, so if the client has paid, and you're business with them is finished, remember to set their client status to 'old' so the deadline field doesn't show `OVERDUE`.
```
Comment (JunLongling) [inline docs/UserGuide.md:156]: Typo – Minor grammatical issue in the text.

Current: " so if the client has paid, and `you're` business with them is finished..."

Suggested Correction: "so if the client has paid, and `your` business with them is finished..."

**Review by JunLongling — COMMENTED** [review]
Overall, the update is clear and well-structured, with only a minor grammatical issue to correct. Once this minor fix is made, the change can be approved. Great work! 🎉

File: docs/index.md [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**Clientele+ is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you are interested in using Clientele+, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
+* If you are interested about developing Clientele+, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```
Comment (Dinoman44) [inline docs/index.md:13]:
```suggestion
* If you are interested in developing Clientele+, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```

**Review by Dinoman44 — COMMENTED** [review]
Some minor nits, otherwise LGTM

File: docs/UserGuide.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS d/DEADLINE [t/TAG]…​
 * `NAME` is case-insensitive. `John Doe` and `joHN dOE` are considered same clients, but name is stored in the same case as the input (so `John Doe` is stored as `John Doe` and `JOHN Doe` is stored as `JOHN Doe`).
 * `PHONE_NUMBER` should be **numeric** digits, may include “-” or spaces. Example: `555-1234` or `555 1234`.
 * `EMAIL`  Standard email format: “user@example.com”.
-* `PAYMENT STATUS` Acceptable values are `pending`, `partial`, `paid`, `late`. Case-insensitive.
+* `PAYMENT STATUS` Acceptable values are `pending`, `partial`, `paid`, `late`. Case-insensitive. These values are displayed as an icon with the colours orange, yellow, green and red respectively.
```
Comment (rahula1008) [inline docs/UserGuide.md:111]: By any chance, could it be a good idea to give some examples of what the colours may look like/the situations you may use each payment status in?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -120,8 +120,9 @@ Examples:
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 
-1. The deadline could be the date you need to submit the project to the client
-2. The deadline could be the date by which you need to follow up with a potential client
+1. The deadline could be the date you need to submit the project to the client <br>
+2. The deadline could be the date by which you need to follow up with a potential client <br>
+3. You may hover over the Payment Status icon to view the Payment Status as text
```
Comment (Dinoman44) [inline docs/UserGuide.md:125]:
```suggestion
<br>1. The deadline could be the date you need to submit the project to the client. <br>
2. The deadline could instead be the date by which you need to follow up with a potential client. <br>
3. You may hover over the Payment Status Icon to view the payment status as text.
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -152,7 +153,7 @@ Examples:
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 
-1. The deadline field will show an `[OVERDUE]` tag if the deadline has passed and the client status is still `active`, so if the client has paid, and you're business with them is finished, remember to set their client status to 'old' so the deadline field doesn't show `[OVERDUE]`.
+1. The deadline field will show an `OVERDUE` label if the deadline has passed and the client status is still `active`, so if the client has paid, and your business with them is finished, remember to set their client status to 'old' so the deadline field doesn't show `OVERDUE`.
```
Comment (Dinoman44) [inline docs/UserGuide.md:156]:
```suggestion
The deadline field will show an `OVERDUE` label if the deadline has passed and the client status is still `active`, so if the client has paid, and your business with them is finished, remember to set their client status to 'old' so the deadline field doesn't show `OVERDUE`.
```

**Review by Dinoman44 — COMMENTED** [review]
Some minor nits, otherwise looks good.

**Review by Dinoman44 — APPROVED** [review]

**Pull request merged**